lang/cs/Org.Apache.REEF.Driver/Bridge/Events/ActiveContext.cs

```diff
@@ -44,48 +43,35 @@ public ActiveContext(IActiveContextClr2Java clr2Java)
         }
 
         [DataMember]
-        public string InstanceId { get; set; }
+        public string InstanceId { get; private set; }
+
+        private IActiveContextClr2Java Clr2Java { get; set; }
```

Comment:
Private properties seem kind of strange to me. Why not just use a private variable?